### ISSUE TITLE
How to use WADGEN?

### ISSUE THREAD

**Issue by htv04** [WiiDatabase/PyNUSD#4]
First, I just want to say this script is amazing! I'm currently using it for a project I'm working on for Linux, since there isn't a Linux version of Sharpii. However, there isn't a lot of documentation for how to use WADGEN and implement it into a script. For example, say I download the Forecast Channel files from the NUS and decrypt them using PyNUSD, then apply a patch to one of the decrypted files. How would I pack the files into a WAD using WADGEN?

**Comment by Brawl345** [comment]
Current WADGEN is not really good to use, it's more made for PyNUSD but it can be used as a lib. There is a `rewrite` branch, but it can't pack WADs yet (and is missing NUS functionality).

You can use the `WADGEN.WADMaker` class to open a folder and pack it as WAD: https://github.com/WiiDatabase/PyNUSD/blob/master/WADGEN.py#L1104

Like so:
```python
wad = WADGEN.WADMaker("path/to/my/files")
wad.encrypt_file("00000007")  # for example when you patched the "00000007.app", you can do this multiple times of course
wad.dump("my-new-wad.wad")
```

It's a bit clunky to use sadly :/ And the WAD format is weird af.

**Comment by htv04** [comment]
Okay, thanks!
